File: 백준/Silver/2563. 색종이/색종이.java
import java.util.*;

public class Main {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        
        int[][] canvas = new int[100][100]; // 전체 좌표 도화지
        
        int N = sc.nextInt();
        
        for(int i = 0; i<N; i++)
        {
        	int x = sc.nextInt(); // 색종이의 왼쪽 아래 모서리 좌표 입력 받음
        	int y = sc.nextInt();
        	
        	for(int j = x; j<x+10; j++)
        	{
               for(int k = y; k<y+10; k++) {
            	   canvas[j][k] = 1;
               }
        	}
        	}
        
        int area = 0; //색종이가 포함되어 있는 구역의 넓이
        
        for(int i = 0; i<100; i++)
        {
        	for(int j = 0; j<100; j++)
        	{
        		if(canvas[i][j] == 1) {
        			area++;
        		}
        	}
        }
        
        System.out.println(area);
    }
}

File: 백준/Silver/2563. 색종이/색종이.py
N = int(input())
canvas = [[0] * 100 for i in range(100)]  # 전체 좌표 도화지

for i in range(N):
    x, y = map(int, input().split())  # 색종이의 왼쪽 아래 모서리 좌표 입력 받음

    for j in range(x, x + 10):
        for k in range(y, y + 10):
            canvas[j][k] = 1

area = sum(row.count(1) for row in canvas)  # 색종이가 포함되어 있는 구역의 넓이

print(area)